s3transfer/utils.py

```diff
@@ -439,8 +439,8 @@ def enable_callback(self):
     def disable_callback(self):
         self._callbacks_enabled = False
 
-    def seek(self, where):
-        self._fileobj.seek(self._start_byte + where)
+    def seek(self, where, whence=0):
```

Comment:
Thanks Kyle.  Yes, that does make sense.  Can I leave it to you to implement that, given your familiarity with the codebase and my lack thereof?  (I'll be happy to verify that your implementation still fixes the zero-length files problem).